src_py/__init__.py

```diff
@@ -332,6 +332,14 @@ def Overlay(format, size):  # pylint: disable=unused-argument
 except (ImportError, IOError):
     pass
 
+# this internal module needs to be included for dependency
```

Comment:
Is it only used for tests or needed for dependency finder? Or both?

What kind of dependencies are behind this?